cfme/tests/middleware/test_middleware_server.py

```diff
@@ -72,10 +74,115 @@ def test_server_details(provider):
+            break
+    assert server, 'Wildfly server is not found in servers list'
+    server.reload_server()
+    flash.assert_success_message('Reload initiated for selected server(s)')
+    # enable when HWKINVENT-185 is fixed
+    # _check_server_stopped(server)
+    sleep(10)
+    _check_server_running(server)
+
+
+def test_hawkular_reload(provider):
+    """Tests Hawkular server itself reload operation message on UI
+
+    Steps:
+        * Get servers list from UI
+        * Chooses Hawkular server from list
+        * Clicks 'Reload Server' operation
+        * Checks that notification message is shown
+        * Checks that server status is running in UI, in DB and in MGMT.
+        * Clicks 'Stop Server' operation
+        * Checks that notification message is shown
+        * Checks that server status is running in UI, in DB and in MGMT.
+    """
+    server_list = MiddlewareServer.servers(provider=provider)
+    for _server in server_list:
+        if _server.product == 'Hawkular':
+            server = _server
+            break
+    assert server, 'Hawkular server is not found in servers list'
+    server.reload_server()
+    flash.assert_success_message('Not reloading Hawkular server')
+    _check_server_running(server)
+    server.stop_server()
+    flash.assert_success_message('Not stopping Hawkular server')
+    _check_server_running(server)
+
+
+''' enable when HWKINVENT-185 is fixed
+def test_server_stop(provider):
+    """Tests server reload operation on UI
+
+    Steps:
+        * Get servers list from UI
+        * Chooses Wildfly server from list
+        * Clicks 'Stop Server' operation
+        * Checks that server status is not running in UI, in DB and in MGMT.
+        * Clicks 'Start Server' operation
+        * Waits for some time
+        * Checks that server status is running in UI, in DB and in MGMT.
+    """
+    server_list = MiddlewareServer.servers(provider=provider)
+    for _server in server_list:
+        if _server.product != 'Hawkular':
+            server = _server
+            break
+    assert server, 'Wildfly server is not found in servers list'
+    server.stop_server()
+    flash.assert_success_message('Stop initiated for selected server(s)')
+    _check_server_stopped(server)
+    sleep(10)
+    server.start_server()
+    sleep(10)
+    _check_server_running(server)
+    '''
+
+
 def _get_servers_set(servers):
     """
     Return the set of servers which contains only necessary fields,
     such as 'feed', 'provider.name', 'name' and 'product'
     """
     return set((server.feed, server.provider.name, server.name, server.product)
                for server in servers)
+
+
+def _check_server_stopped(server):
+    stopped_db = False
+    stopped_mgmt = False
+    for _ in range(0, 10):
+        stopped_db = not server.is_running(method='db')
+        stopped_mgmt = not server.is_running(method='mgmt')
+        if not stopped_db or not stopped_mgmt:
+            sleep(0.5)
```

Comment:
@psav so have this line instead of whole body"
"wait_for(lambda: not server.is_running(method='db') and not server.is_running(method='mgmt'), delay=0.5, num_sec=600)"